app/Services/Api/ApiPermissionService.php

```diff
@@ -60,4 +63,43 @@ public function saveEntity($request, Entity $entity)
         }
         return $model;
     }
+
+    /**
+     * @param $request
+     * @param Campaign $campaign
+     */
+    public function entityPermissionTest($request, Campaign $campaign)
+    {
+        $previousUser = 0;
+
```

Comment:
Missing declaring `$permissionTest` as an array at the beginning of the function